src/widgets/Menu/Avatar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { Profile } from "./types";
+import NoProfileAvatar from "./icons/NoProfileAvatar";
+
+interface AvatarProps {
+  profile?: Profile;
+}
+
+const StyledAvatar = styled.div`
+  margin-left: 8px;
+
+  img {
+    border-radius: 50%;
+  }
+`;
+
+const Avatar: React.FC<AvatarProps> = ({ profile }) => {
+  if (!profile) {
+    return null;
+  }
```

Comment:
This allows the possibility of not including the avatar at all for whatever reason.

Reply:
This logic should be in the component that render `Avatar`